eslint.config.mjs

```diff
@@ -513,6 +513,10 @@ export default tseslint.config(
       'react/jsx-no-target-blank': 'off',
       'react/no-unescaped-entities': 'off',
       'react-hooks/exhaustive-deps': 'warn', // TODO: enable it later
+      /* This rule
+        (1) causes false reports because it reads from `typeParameters` rather than `typeArguments`, and
+        (2) is unnecessary in TypeScript components */
```

Comment:
For (1), if it's a transient problem, we'd want to link to the upstream issue (https://github.com/jsx-eslint/eslint-plugin-react/issues/3602). Vague comments tend to get out of date and don't help new readers understand the issue well.

For (2), yup, that's going to last beyond (1) - so I'd say it makes sense to disable the rule here always anyway.

```suggestion
```